src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/MigrationResource.kt

```diff
@@ -20,38 +19,13 @@ import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.HistoryFi
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.MigrationHistoryService
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.MigrationType
 
-@RestController
+@RestController("prisonPersonMigrationResource")
 @RequestMapping("/migrate/prisonperson", produces = [MediaType.APPLICATION_JSON_VALUE])
-class PrisonPersonMigrationResource(
-  private val prisonPersonMigrationService: PrisonPersonMigrationService,
+@Tag(name = "prison-person-migration-resource")
+class MigrationResource(
+  private val migrationService: MigrationService,
   private val migrationHistoryService: MigrationHistoryService,
 ) {
-  @PreAuthorize("hasRole('ROLE_MIGRATE_PRISONPERSON')")
-  @PostMapping("/physical-attributes")
```

Comment:
Anything specific to physical attributes has been moved up into the physical attributes package.